system-container/README.md

```diff
@@ -0,0 +1,60 @@
+# System Container support
+
+A system container runs using runC instead of Docker and it is managed by the
+[atomic](https://github.com/projectatomic/atomic/) tool.  As it doesn't require
+Docker to run, a container that makes changes to Docker's configuration/status
+can be run on a sytem that is itself a target/part of the inventory.
+
+The files under [`exports/`](exports/) are needed to run the the built images as
+an [Atomic system container](http://www.projectatomic.io/blog/2016/09/intro-to-system-containers/):
+
+* `config.json.template` is the configuration file to run the runC container.
+
+* `manifest.json` defines various settings for the system container, such as the
+  default values to use for the installation.
+
+* `service.template` is the template for the systemd service.
+
+* `tmpfiles.template` is the template for systemd-tmpfiles.
+
+The base playbook2image container image already contains all the required files
+to run as a system container, so no specific action needs to be taken to build a
+system container-ready image based on playbook2image.
+
+Once a container image is built (e.g. `my/image`) you can import it into the OSTree storage:
+
+    atomic pull --storage ostree docker:my/image:latest
+
+## Running an image as a System Container
+
+To run the playbook `myplaybook.yml` from `myimage` as a system container:
+
+```sh
+atomic install --system \
+	   --set INVENTORY_FILE=$(pwd)/myinventory \
+	   --set PLAYBOOK_FILE=myplaybook.yml \
+	   myimage
+systemctl start myimage
+```
+
+And to finally cleanup the container:
+
+    atomic uninstall myimage
+
+*NOTE:* after the next release of the `atomic` command that includes [PR 952](https://github.com/projectatomic/atomic/pull/952) we will add an `atomic.run=once` label and the `systemctl` invocation or the cleanup/uninstall won't be necessary.
```

Comment:
Yes it was, but just recently (last week) there was a release of the `atomic` tool with the changes.

Now that this is actually available I will do some actual testing :) and update